src/command_modules/azure-cli-servicebus/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+
+0.2.4
++++++
+* az servicebus list commands error are handled in common structure
+
```

Comment:
What do you think of rephrasing the change as: "az servicebus list commands' errors are now handled in the typical way"? Also did you rerun the tests you modified live?

Otherwise the PR looks good to me. What do you think @tjprescott?

Reply:
updated the History file and I have rerun the test live.

Reply:
Thanks for the updates!

Reply:
@tjprescott informed me that if removing the 404 exception handlers is a breaking change, then we should bump up the minor version number and note that this would be a breaking change.